data/helpers/recipeDb.js

```diff
@@ -0,0 +1,18 @@
+const db = require('../dbConfig');
+
+module.exports = {
+  getRecipes() {
```

Comment:
Good job creating your helper file. (There's got to be a better name than helpers, I just have always seen them called helpers and couldn't find anything other than helper function in my google search)

Reply:
Just makes for cleaner code, breaking the helpers into their own file.